lib/pages/chat/events/html_message.dart

```diff
@@ -77,7 +77,8 @@ class HtmlMessage extends StatelessWidget {
       shrinkToFit: true,
       maxLines: maxLines,
       onLinkTap: (url) => UrlLauncher(context, url.toString()).launchUrl(),
-      onPillTap: (url) => UrlLauncher(context, url).launchUrl(),
+      onPillTap: chatController?.onPillTap ??
```

Comment:
onTagTap

Reply:
Fixed in [336b77c](https://github.com/linagora/twake-on-matrix/pull/1007/commits/336b77c1427b32d19e2b3cf6f5cdf98d20b0ce62)